bundle/language_en.properties

```diff
@@ -17,10 +17,7 @@
 # Common dialog titles
 #######################
 angal.common.error.title                                                         = Error
-angal.common.warning.title                                                       = Warning
 angal.common.info.title                                                          = Information
-angal.common.question.title                                                      = Question
-angal.common.note.title                                                          = Note
```

Comment:
`angal.common.note.title` can be restored because there are several part in the application using "Note" with the same meaning

Reply:
Done